internal/applicationsnapshot/report_test.go

```diff
@@ -61,7 +62,21 @@ func Test_Report(t *testing.T) {
       ]
     }
   `
+	var components []Component
+	for _, component := range snapshot.Components {
+		c := Component{
+			Violations: []output.CheckResult{},
+			Success:    true,
+		}
+		c.Name, c.ContainerImage = component.Name, component.ContainerImage
+		components = append(components, c)
+	}
 
-	report, _ := Report(snapshot)
+	report, _, success := Report(components)
 	assert.JSONEq(t, expected, report)
+	assert.True(t, success)
+
+	components = append(components, Component{Success: false})
+	report, _, success = Report(components)
```

Comment:
I guess we forgot to assert the `report` here?

Reply:
Yup, added